exporters/failure/collector_github.py

```diff
@@ -125,7 +125,11 @@ def search_issues(self) -> list[TrackerIssue]:
             for issue in all_issues:
                 is_bug = False
                 labels = issue["labels"]
-                is_bug = any(label for label in labels if pelorus.get_github_issue_label() in label["name"])
+                is_bug = any(
+                    label
+                    for label in labels
+                    if pelorus.get_github_issue_label() in label["name"]
+                )
```

Comment:
I just realized-- this means if someone's label is `bug`, this will catch labels that are, for example, `debug`. Or even `not-a-bug`. Should we use an exact match instead?

Reply:
true..  
We could change that to be an exact match of the "bug" label.  I'm not 100% sure an exact match is right thing tbh.

For the metric to be record a combination of the two labels is required.  A metric needs the bug label and APP_LABEL so I think we're pretty safe for cases where "debug" is used.   I'm not wedded to either approach just saying a user has to be very intentional w/ labels to get the metric recorded.